src/pages/SupplierRequisitionPage.js

```diff
@@ -354,10 +354,11 @@ const mapDispatchToProps = (dispatch, ownProps) => {
 const mapStateToProps = state => {
   // TODO: add indicator toggle flag to page state.
   const { pages } = state;
-  const { showIndicators, indicatorColumns } = pages[ROUTES.SUPPLIER_REQUISITION];
+  const { showIndicators, indicatorColumns, indicatorRows } = pages[ROUTES.SUPPLIER_REQUISITION];
   if (showIndicators) {
     return {
       ...pages[ROUTES.SUPPLIER_REQUISITION],
+      data: indicatorRows,
```

Comment:
Like the idea, but at this stage I personally find this way a bit easier to follow. Mostly just because its a bit more explicit in indicating the subset of props that are overwritten by indicators functionality.